micrometer-tracing-bridges/micrometer-tracing-bridge-brave/src/main/java/io/micrometer/tracing/brave/bridge/BraveTracer.java

```diff
@@ -44,6 +44,9 @@ public BraveTracer(brave.Tracer tracer, CurrentTraceContext context, BaggageMana
         this.tracer = tracer;
         this.braveBaggageManager = braveBaggageManager;
         this.currentTraceContext = context;
+        if (braveBaggageManager instanceof BraveBaggageManager) {
+            ((BraveBaggageManager) braveBaggageManager).setTracer(this);
```

Comment:
Because of this we don't need to modify auto-configuration in Boot, right?

Reply:
Correct, we're not adding additional constructors.